tests/suite-test-1.2.rc

```diff
@@ -0,0 +1,53 @@
+# SYNTAX TEST "source.cylc" "Comments"
+# See https://github.com/cylc/cylc-flow/issues/2752
+
```

Comment:
Looks like you've separated your tests into logical blocks - and it's a great framework to read.

I think that the name of the test files should give more of a clue about what they are testing.
Failing (or in addition to) that, I'd like a little more comment in the top of the file.

```suggestion
# Testing that comments are correctly identified.
```

Reply:
Yes I did wonder the same